CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+## <small>1.15.5 (2018-12-06)</small>
+
+* changelog ([4cf4f32](https://github.com/bigcommerce/stencil-cli/commit/4cf4f32))
```

Comment:
Shouldn't this be generated when we release 1.15.5 ?

Reply:
I released 1.15.5